ballerina/tests/08-procedure-call-test.bal

```diff
@@ -67,6 +67,17 @@ isolated function beforeProcCallFunc() returns sql:Error? {
         id, col_number, col_float, col_binary_float, col_binary_double)
         VALUES (1, 2147483647, 21474.83647, 21.47483647, 21474836.47)`
     );
+
+    result = check dropTableIfExists("CallComplexTypes");
+    result = check oracledbClient->execute(`CREATE TABLE CallComplexTypes(
+        id NUMBER,
+        col_xml XMLType,
+        PRIMARY KEY (id)
+        )`
```

Comment:
Do not create test tables on the fly. Lets initiate it through the starting scripts

Reply:
Will update those test tables creations in a separate PR.

Issue was created to track the task. https://github.com/ballerina-platform/ballerina-standard-library/issues/1729